src/utils.ts

```diff
@@ -80,9 +80,22 @@ function isObjectEmpty (object: Object): boolean {
   return Object.keys(object).length === 0;
 }
 
+function replaceAll (input: string, search: string, replace: string) {
+  return input.split(search).join(replace);
```

Comment:
Could this be used instead? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll

Reply:
This was added in August 2020, and is not part of ES6

Reply:
@RLRabinowitz Could you please take a look at this [tutorial](https://www.tutorialspoint.com/typescript/typescript_string_replace.htm)
Can we use it here?